contracts/extensions/MultisigPermissions.sol

```diff
@@ -503,6 +503,15 @@ contract MultisigPermissions is ColonyExtensionMeta, ColonyDataTypes, GetActionS
 
   function validateUserPermissions(uint256 _permissionDomainId, uint256 _motionId) internal view {
     bytes32 userPermissions = getUserRoles(msgSender(), _permissionDomainId);
+    Motion storage motion = motions[_motionId];
+
+    if ((motion.requiredPermissions & bytes32(1 << uint256(ColonyRole.Architecture))) != 0) {
```

Comment:
Regarding the failing test, my first thought would be to expand this require to explicitly check the motion signature against the signature which use the `archSubdomain` modifier. That could be done here, or through the "dominant signatures" concept you've introduced elsewhere, or potentially both places.

Reply:
It's possible the solution I've implemented here is too cute, but see what you think.

The reason that `archSubdomain` exists at all is because we allow `root` and `architecture` to call those (permission-setting) functions in different circumstances, but that means that `requiredPermissions` is `1 << ColonyRole.Root & 1 << ColonyRole.Architecture` for those functions, and just `1 << ColonyRole.Architecture` for the other functions (those that affect domains / domain structure). That means this check can be keyed off of `requiredPermissions` by being more precise.